.maven.xml

```diff
@@ -0,0 +1,24 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
```

Comment:
This change has been pushed back from the last version [here](https://github.com/onfido/onfido-java/blob/master/.maven.xml) since needed by the workflow to publish the package. It's going to be ignored from the files that are automatically overridden on library generation.